sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/cosmos/properties/AzureCosmosProperties.java

```diff
@@ -97,7 +92,7 @@
     /**
      * Connection mode to be used by the client in the Azure Cosmos DB database service.
      */
-    private ConnectionMode connectionMode;
+    private ConnectionMode connectionMode = ConnectionMode.DIRECT;
```

Comment:
The test properties should be updated in sync, com.azure.spring.service.implementation.cosmos.TestAzureCosmosHttpProperties#connectionMode